apps/io-wallet-user-func/local.settings.json.example

```diff
@@ -45,6 +45,7 @@
     "PidIssuerApiRequestTimeout": "PID_ISSUER_API_REQUEST_TIMEOUT",
     "HttpRequestTimeout": "HTTP_REQUEST_TIMEOUT",
     "SkipSignatureValidation": "SKIP_SIGNATURE_VALIDATION",
+    "APPLICATIONINSIGHTS_CONNECTION_STRING": "APPLICATIONINSIGHTS_CONNECTION_STRING",
```

Comment:
keep in mind that if you use the AI sdk you usually want to avoid to start the default AI agent which starts automatically if you use this env, see https://pagopa.github.io/dx/docs/articles/azure-tracing#integration-with-app-service